modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -300,22 +350,26 @@ Diagram.defaultProps = {
 			selectedColor: '0B5FFF',
 			useNative: true,
 		},
-		defaultRadius: 15,
+		defaultRadius: 2,
 	},
 	pins: [],
+	pinsEndpoint:
+		'http://localhost:8080/o/headless-commerce-admin-catalog/v1.0/',
+	productId: 44206,
```

Comment:
I'd suggest storing this id in a constant at the top of the file

Reply:
This id can't be used, if it will be used the api calls will be wrong unless you have a product with the exact same id. It was only for my testing purposes, but this value must be evaluated ever from props, so we should get ever the correct id to call

Reply:
anyway i can remove it, it's less important that pinsEndpoint. If it's missing could be undefined (always in static env), inside lf module it will be evaluated for pure with a productId